module/data/item/race.mjs

```diff
@@ -0,0 +1,73 @@
+import SystemDataModel from "../abstract.mjs";
+import { AdvancementField, IdentifierField } from "../fields.mjs";
+import ItemDescriptionTemplate from "./templates/item-description.mjs";
+
+/**
+ * Data definition for Race items.
+ * @mixes ItemDescriptionTemplate
+ *
+ * @property {string} identifier       Identifier slug for this race.
+ * @property {object[]} advancement    Advancement objects for this race.
+ * @property {number} darkvision       Darkvision distance.
+ * @property {object} movement
+ * @property {number} movement.burrow
+ * @property {number} movement.climb
+ * @property {number} movement.fly
+ * @property {number} movement.swim
+ * @property {number} movement.walk
+ * @property {string} movement.units   Units used to measure movement.
+ * @property {object} type
+ * @property {string} type.value       Creature type (e.g. "humanoid", "construct").
+ * @property {string} type.subtype     Additional type data (e.g. "elf" for Eladrin)
+ */
+export default class RaceData extends SystemDataModel.mixin(ItemDescriptionTemplate) {
+  /** @inheritdoc */
+  static defineSchema() {
+    return this.mergeSchema(super.defineSchema(), {
+      identifier: new IdentifierField({label: "DND5E.Identifier"}),
+      advancement: new foundry.data.fields.ArrayField(new AdvancementField(), {label: "DND5E.AdvancementTitle"}),
+      darkvision: new foundry.data.fields.NumberField({min: 0, integer: true, label: "DND5E.SenseDarkvision"}),
```

Comment:
Would it be worthwhile to generalize that the `race` contains a `senses` data structure that then translates onto `actor.system.attributes.senses`? Might certain races have something other than darkvision that we would need to record? Perhaps not but it could accommodate for homebrew while also reusing an existing data structure that we have.